script/Ciel/Day 7/07_01_CIEL07_1_1.txt

```diff
@@ -33,5 +33,6 @@ Something in what he just said tugs at the back of my mind, but it isn't enough
 -- Page 86, Offset 15009.
 -- 俺は絶好のチャンスを棒に振って、男友達と仲良くランチに行くのだった。
 // 男 split off of 友達 to emp lack of Ciel without having Shiki say 'broski' or something
+// lads lunch? seems too british to me though i dont have a good alternative at the moment
```

Comment:
It's alliterative and passed the Ben sniff test /shrug

Reply:
ok works for me